config/globals.js

```diff
@@ -112,6 +112,10 @@ const globals = {
 	ETHERPAD_OLD_PAD_DOMAIN: process.env.ETHERPAD_OLD_PAD_DOMAIN,
 	ETHERPAD_COOKIE__EXPIRES_SECONDS: process.env.ETHERPAD_COOKIE__EXPIRES_SECONDS,
 	ETHERPAD_ETHERPAD_COOKIE_RELEASE_THRESHOLD: process.env.ETHERPAD_COOKIE_RELEASE_THRESHOLD,
+
+	// silent error
+	SILENT_ERROR_ENABLED: process.env.SILENT_ERROR_ENABLED || false,
```

Comment:
please use Configuration.get('SILENT_ERROR_ENABLED') instead of globals

Reply:
Done